src/OrchardCore.Modules/OrchardCore.Users/Controllers/AdminController.cs

```diff
@@ -561,6 +559,56 @@ public async Task<IActionResult> EditPassword(EditPasswordViewModel model, strin
         return View(model);
     }
 
+    [HttpPost]
+    public async Task<IActionResult> Enable(string id)
+    {
+        if (await _userManager.FindByIdAsync(id) is not User user)
+        {
+            return NotFound();
+        }
+
+        if (!await _authorizationService.AuthorizeAsync(User, UsersPermissions.EditUsers, user))
+        {
+            return Forbid();
+        }
+
+        if (await _userService.EnableAsync(user))
+        {
+            await _notifier.SuccessAsync(H["User enabled successfully."]);
```

Comment:
```suggestion
            await _notifier.SuccessAsync(H["User account was successfully enabled"]);
```

Reply:
I'm using the same message for the unlock action

Reply:
I understand. It's grammatically incorrect. Please take the suggested verbiage instead.